R/coerce_to_messydate.R

```diff
@@ -319,6 +348,46 @@ extract_from_text <- function(v) {
   out
 }
 
+written_month <- function(dates) {
```

Comment:
This is really neat. I guess we will need to test for false positives and maybe consider other languages in the future?

Reply:
There is an issue currently open for the expansion/improvements of this function. I added these points as comments in that issue, thank you.